### PR TITLE
Script to enable interfaces inorder to send/receive LLDP

### PR DESCRIPTION
**Pull request by mtannous**
This script iterates through all the interfaces in the microkernel and sets their default IP address, netmask, gateway. Once all the interfaces are set, then the script makes sure that all interfaces are up.

Referencing this pull request: 
https://github.com/RackHD/on-tasks/pull/367

### CODE REVIEW

**Comment by benbp** [comment]
Please squash and rebase the commits.

**Comment by benbp** [comment]
Requires https://github.com/RackHD/on-http/pull/512

**Comment by mtannous** [comment]
Thanks, just did.

File: data/templates/set_interfaces.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+string= ""
+interfaces=[]
+for dirname, dirnames, filenames in os.walk('/sys/class/net/'):
+    for subdirname in dirnames:
+        if subdirname != "lo":
+            interfaces.append(subdirname)
+            ip= 100+count
+            string += "auto " + subdirname
+            string += "\niface " +  subdirname +" inet static"
+            string += "\naddress 10.0.0." + str(ip)
```
Comment (yyscamper) [inline data/templates/set_interfaces.py]: @mtannous : please add some comment, so other developer will be aware of this is not a magic IP address, actually any one is OK.

**Comment by yyscamper** [comment]
jenkins: ok to test

File: data/templates/set_interfaces.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import os
+import time
+count = 0
+string= ""
+interfaces=[]
+for dirname, dirnames, filenames in os.walk('/sys/class/net/'):
+    for subdirname in dirnames:
+        if subdirname != "lo":
+            //Here we are assigning an arbitrary IP which could be a different one. This is just to get the interface up
```
Comment (yyscamper) [inline data/templates/set_interfaces.py]: @mtannous : for python, the comment symbol is `#` not `//`

**Comment by nucklehead** [comment]
Can this be merged please :-)

**Comment by changev** [comment]
test this please

**Comment by changev** [comment]
test this please

**Pull request merged**